packages/target-decisioning-engine/types/TelemetryEntry.d.ts

```diff
@@ -0,0 +1,26 @@
+export interface TelemetryRequest {
```

Comment:
As discussed, probably it's too early for these changes, let's remove them until we  have a chance to talk about Phase 2.